### DISCUSSION TITLE
Question: Time/Date Picker Highlight height?

### DISCUSSION THREAD

**Discussion by mdtauk** [microsoft/microsoft-ui-xaml#7482] [Q&A]
When the changes to the Date/Time Picker was proposed, I had asked if the highlighted area height would remain 40px or be reduced to 32px to match the size of the Form Control which when tapped, opens the flyout control.

It was said the design team had agreed and the height would change, but now the new control is in the repo, the 40px height persists.  Was this a conscious decision, or could it have been overlooked?

![image](https://user-images.githubusercontent.com/7389110/117619969-d77d6800-b167-11eb-8fa6-2b8cb8459cfb.png)

**Comment by StephenLPeters** [comment]
@chigy FYI
